tests/mockserver.py

```diff
@@ -206,7 +206,7 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_value, traceback):
         self.proc.kill()
-        self.proc.wait()
+        self.proc.communicate()
         time.sleep(0.2)
```

Comment:
I commented out this line locally and it worked, perhaps it was an attempt to prevent this kind of issues and it's not necessary anymore if we use `communicate`?